config.yaml.sample

```diff
@@ -4,5 +4,4 @@ default:
     Base64HighEntropyString: 4.5
```

Comment:
Do we have config files in the core, client code?

Reply:
You're right, I missed this one -- deleted.